src/Controllers/UserController.js

```diff
@@ -0,0 +1,143 @@
+import { randomString } from '../cryptoUtils';
+import { inflate } from '../triggers';
+import AdaptableController from './AdaptableController';
+import MailAdapter from '../Adapters/Email/MailAdapter';
+
+var DatabaseAdapter = require('../DatabaseAdapter');
+
+export class UserController extends AdaptableController {
```

Comment:
I have a feeling the `UserController` has a decent chance of requiring multiple adapters (eg. verify user phone number). I suspect this `AdaptableController` approach might not scale very well to more complex scenarios.